.pipelines/build-model-container-image.Jenkinsfile

```diff
@@ -40,19 +41,17 @@ pipeline {
             steps {
                 echo "Hello docker image build ${env.BUILD_ID}"
                 checkout scm
-                //checkout([$class: 'GitSCM', branches: [[name: '*/ml_model_uc81']],
-                //    userRemoteConfigs: [[url: 'https://github.com/Merlion-Crew/MLOpsPython.git/']]])
 
                 sh '''
-                    conda env create --file ./diabetes_regression/ci_dependencies.yml --force 
+                    conda env create --name $DEFAULT_CONDA_ENV_NAME --file ./diabetes_regression/ci_dependencies.yml --force 
```

Comment:
Just curious about the `$DEFAULT_CONDA_ENV_NAME`. The `ci_dependencies.yml` was configured to have a name of `mlopspython_ci`, is there any reason why we are changing the conda env name?

Reply:
Swetha gave a comment if the yml file has changed name property's value in the future, it will break the pipeline. It makes sense, so I removed the dependency on environment name by overwriting it with new variable's name, so it should be good if there is any change.